android/app/src/main/AndroidManifest.xml

```diff
@@ -16,6 +16,7 @@
             android:name=".MainActivity"
             android:exported="true"
             android:label="Subletr"
+            android:windowSoftInputMode="adjustPan"
```

Comment:
This line fixes this issue with the keyboard pushing text up
I think this just makes the screen scrollable when the keyboard pops up, which I prefer personally

![image](https://github.com/andr-dev/CS446/assets/46252938/ba1e5b7c-c12e-42cb-afbb-c91e13879692)